kafka-ui-react-app/src/components/Connect/List/List.tsx

```diff
@@ -22,7 +21,21 @@ const List: React.FC = () => {
 
   const columns = React.useMemo<ColumnDef<FullConnectorInfo>[]>(
     () => [
-      { header: 'Name', accessorKey: 'name' },
+      {
+        header: 'Name',
+        accessorKey: 'name',
+        // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
In order to get rid of this eslint error , you can make this another component and put the component itself as a cell

`{ header: 'Name', accessorKey: 'name' ,cell:ComponentNameHere},` cause they already receive this properties and it will be uniformly with the project structure of defining columns